# Rapid evolution of pearl oyster shell matrix proteins with repetitive, low-complexity domains - Supplementary Data

This GitHub repository provides the supplementary data referenced in the publication cited below.

## How to use and cite these files 

All files are publicly available and can be used for further research or other applications. However, if you utilize these resources in your work, we kindly request that you cite our original publication.

**Rapid evolution of pearl oyster shell matrix proteins with repetitive, low-complexity domains.** Carmel McDougall, Felipe Aguilera and Bernard M. Degnan. *Journal of the Royal Society Interface* **10**:20130041 (2013). https://doi.org/10.1098/rsif.2013.0041

**Abstract**

The lysine (K)-rich mantle protein (KRMP) and shematrin protein families are unique to the organic matrices of pearl oyster shells. Similar to other proteins that are constituents of tough, extracellular structures, such as spider silk, shematrins and KRMPs, contain repetitive, low-complexity domains (RLCDs). Comprehensive analysis of available gene sequences in three species of pearl oyster using BLAST and hidden Markov models reveal that both gene families have large memberships in these species. The shematrin gene family expanded before the speciation of these oysters, leading to a minimum of eight orthology groups. By contrast, KRMPs expanded primarily after speciation leading to species-specific gene repertoires. Regardless of their evolutionary history, the rapid evolution of shematrins and KRMPs appears to be the result of the intrinsic instability of repetitive sequences encoding the RLCDs, and the gain, loss and shuffling of other motifs. This mode of molecular evolution is likely to contribute to structural characteristics and evolvability of the pearl oyster shell. Based on these observations, we infer that analogous RLCD proteins throughout the animal kingdom also have the capacity to rapidly evolve and as a result change their structural properties.

## Author contact

- Carmel McDougall c.mcdougall@uq.edu.au (senior first-author)
- Felipe Aguilera f.aguilera@uq.edu.au
- Bernie Degnan b.degnan@uq.edu.au (corresponding author)

## How to download data files

The files in this repository are ready for use. Simply click on the file, and you will be directed to the GitHub webpage to save it to your device.

## Index of data file contents

[01-Supplementary_Tables_Figures_Sequences.doc](https://github.com/faguil/RLCD-Evolution/blob/main/01-Supplementary-Tables-Figures-Sequences.doc) contains tables, figures, and Shematrin/KRMP sequences used in this study.

- **Supplementary table S1.** Genomic location of *Pinctada fucata* shematrin genes.
- **Supplementary table S2.** Genomic location of probable alleles of *Pinctada fucata* shematrin genes.
- **Supplementary table S3.** Revised names of previously published shematrin sequences.
- **Supplementary table S4.** Genomic location of *Pinctada fucata* KRMP genes.
- **Supplementary table S5.** Genomic location of probable alleles of *Pinctada fucata* KRMP genes.
- **Supplementary table S6.** Revised names of previously published KRMP sequences.
- **Supplementary figure S1.** Genomic arrangement of *P. fucata* shematrin genes.
- **Supplementary figure S2.** Examples of repeat element variants commonly seen in shematrin and KRMP sequences.
- **Supplementary figure S3.** Alignment of shematrin motifs 1, 2 and 3.
- **Supplementary figure S4.** Phylogenetic relationships of shematrin genes.
- **Supplementary list S1.** Shematrin and KRMP sequences used in these analyses.

[02-KRMP-HMM-Markov-model.txt](https://github.com/faguil/RLCD-Evolution/blob/main/02-KRMP-HMM-Markov-model.txt) contains the profile hidden Markov models (profile HMMs) for the KRMP gene family.

[02-KRMP-HMM-Markov-model.txt](https://github.com/faguil/RLCD-Evolution/blob/main/02-KRMP-HMM-Markov-model.txt) contains the profile hidden Markov models (profile HMMs) for the KRMP gene family.
